src/rpcrawtransaction.cpp

```diff
@@ -808,9 +808,9 @@ UniValue sendrawtransaction(const UniValue& params, bool fHelp)
         throw JSONRPCError(RPC_DESERIALIZATION_ERROR, "TX decode failed");
     uint256 hashTx = tx.GetHash();
 
-    bool fOverrideFees = false;
-    if (params.size() > 1)
-        fOverrideFees = params[1].get_bool();
+    CFeeRate rawMaxFeeRate = CFeeRate(::minRelayTxFee.GetFeePerK() * 10000);
```

Comment:
This should be a separate constant I think - the min relay tx fee changing doesn't necessarily change what constitutes an "absurd" fee. For instance, when we recently upped the min relay fee it was for reasons that shouldn't have impacted the upper bound.

Better would be for this to be a config file setting, shared across the RPC interface and wallet, with some sane default value that gets adjusted as needed.